### DISCUSSION TITLE
How to change the font set ?

### DISCUSSION THREAD

**Discussion by yesfeel** [johnlindquist/kit#1125] [Q&A]
Hello. I don't know the java script. but this scriptkit is very good application for me. 
I have a question. How can i change the font set for integrated terminal of script kit?
I'm using the ligature&nerd font set for the terminal. 
When i using the integrated terminal of script kit, some letters are not displayed properly.

**Comment by johnlindquist** [comment]
@yesfeel It's actually hard-coded to "monospace" in the app. I'll make it configurable in the next release 👍

**Reply by benjamin-guibert** [reply]
Thanks for the awesome work!

Will it be also possible to custom the terminal font? I guess it'd be a valuable feature. ✨

**Reply by johnlindquist** [reply]
@benjamin-guibert Yup, got a build with it working, trying to get it out this weekend

**Reply by zia-hasan** [reply]
is there any update on this? I am using zsh with ohmyzsh and my terminal has special fonts. I think hack nerd font. on the kit terminal I see squares 
<img width="726" alt="Screenshot 2024-01-18 at 12 29 57 AM" src="https://github.com/johnlindquist/kit/assets/13936438/9da04b8d-4541-464e-b43b-3ded05634020">

**Reply by johnlindquist** [reply]
@benjamin-guibert @yesfeel Answering this here too:

https://github.com/johnlindquist/kit/discussions/821#discussioncomment-7523704

**Reply by zia-hasan** [reply]
Fonts still don't render in tmux sessions on the kit terminal. This oould be addressed either by running tmux with utf-8 `tmux -u` or adding this to `.zshrc` before `export ZSH="$HOME/.oh-my-zsh"`  (if you use oh my zsh)(
```
export LC_ALL=en_IN.UTF-8
export LANG=en_IN.UTF-8
```
I tried adding to the tmux config but didn't work for me. Adding the information for other people who ever run into this issue.
